support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -323,7 +323,9 @@ export function CheckoutComponent({
 		.filter(isPaymentMethod)
 		.filter(paymentMethodIsActive);
 
-	const [paymentMethod, setPaymentMethod] = useState<PaymentMethod>();
+	const [paymentMethod, setPaymentMethod] = useState<PaymentMethod | undefined>(
```

Comment:
The type of this useState has been broadened to `PaymentMethod | undefined`. I think that reflects the existing reality (there are already places in the code where we check against it being undefined).